extensions/wikia/CreateNewWiki/FinishCreateWikiController.class.php

```diff
@@ -117,8 +112,15 @@ private function initHeroModule( $mainPageTitle ) {
 		$wikiDataModel->description = $this->params['wikiDescription'];
 		$wikiDataModel->storeInProps();
 		$wikiDataModel->storeInPage();
+	}
 
-		return self::HERO_IMAGE_MAIN_PAGE_MARKUP;
+	/**
+	 * Gets markup for
+	 * @returns string - main page article wiki text
+	 */
+	private function getMoMMainPage( $mainPageTitle ) {
```

Comment:
This is trivial, but I wanted to keep the same level of abstraction in https://github.com/Wikia/app/pull/6483/files#diff-9b3cb789b08a044ed48739f7ef214f42R86 and https://github.com/Wikia/app/pull/6483/files#diff-9b3cb789b08a044ed48739f7ef214f42R88